ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -37,6 +37,14 @@ public interface UtxoLedgerService {
     @Suspendable
     UtxoTransactionBuilder createTransactionBuilder();
 
+    /**
+     * Verifies {@link UtxoLedgerTransaction} in the notary server when using contract-verifying-notary.
```

Comment:
I think this is a bit too specific for an API doc. We shouldn't really say where this will be called, it might be used in other places as well in the future and then this documentation will not make sense then.

Reply:
That's true but I think we still should list what notary can be called and what not to. If this is going to be public API.

Reply:
I agree with Nandor here - we're using it for the notary implementation here, but this can (and probably will) have other use cases, so this comment it not quite right.